src/api-reference/direct-connects/hotel-service-2/Headers.markdown

```diff
@@ -3,20 +3,25 @@ title: Headers
 layout: reference
 ---
 
```

Comment:
Add the pre-release include from the doc template, we'll add it to all the Hotel Service 2 pages. 
{% include prerelease.html %}